monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/commands/local_windows_cmd.js

```diff
@@ -1,10 +1,13 @@
 import {OS_TYPES} from '../OsTypes';
 
 
-export default function generateLocalWindowsCmd(ip, osType) {
+export default function generateLocalWindowsCmd(ip, osType, username) {
   let bitText = osType === OS_TYPES.WINDOWS_32 ? '32' : '64';
-  return `powershell [System.Net.ServicePointManager]::ServerCertificateValidationCallback = {$true};
-   (New-Object System.Net.WebClient).DownloadFile('https://${ip}:5000/api/monkey/download/
-   monkey-windows-${bitText}.exe','.\\monkey.exe');
-   ;Start-Process -FilePath '.\\monkey.exe' -ArgumentList 'm0nk3y -s ${ip}:5000';`;
+  let runAsUser = '';
+    if (username != '')
+      runAsUser = `runas /user:${username} `;
```

Comment:
This doesn't work with powershell. Can you use something like `start powershell -credential ""` and test that it works?